### PR TITLE
MNTOR-2316: US Landing Page Variation when hit 18K OneRep activations

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-2316

# Description
When we hit 18k OneRep activation limit within 30 days, user will be redirected to the waitlist
This PR is only the API portion of the work. There will be a small follow-on PR for the url redirect

# How to test

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/app/functions/server/onerep.ts
Comment (rhelmer) [inline src/app/functions/server/onerep.ts:455]: Hm this would work... I was thinking of having a cron job that did this and populated the database, which we could query from Grafana as well for graphing and alerts. wdyt?

Comment (mansaj) [reply src/app/functions/server/onerep.ts:455]: What would that table look like? Since telegraf does the time series magic automatically, it would just be one entry in the db?

Comment (mansaj) [reply src/app/functions/server/onerep.ts:455]: kind of like subscriber count is just a count(*) that telegraf charts overtime by itself

File: src/app/functions/server/onerep.ts
Comment (pdehaan) [inline src/app/functions/server/onerep.ts:460]: Not sure if we want to extract this into some sort of `isoFromDate()` helper since we're doing a `.substring(0, 10)` in at least 3 places here.
```js
function isoFromDate(date = new Date()) {
  const [dateString] = date.toISOString().split("T", 1);
    return dateString;
}
```

I think we could also use `Array#unshift()` if you want a one-liner.
But might even reduce this to something like this, since the default `date` is `new Date()` so we wouldn't need the `to ? to... : new Date()...` bits:
```js
const fromDateString = isoFromDate(from);
const toDateString = isoFromDate(to);
```

Conversely
```js
const toDateString = (to ?? new Date()).toISOString().split("T", 1).shift();
```

File: src/app/functions/server/onerep.ts
Comment (pdehaan) [inline src/app/functions/server/onerep.ts:456]: Is this called from somewhere yet?

File: src/app/functions/server/onerep.ts
Comment (rhelmer) [reply src/app/functions/server/onerep.ts:455]: I put a simple example in https://github.com/mozilla/blurts-server/pull/3779 - I think the query would need to be a slightly more complicated but not much more (`select count(*) where current >= max` or so)

File: src/app/functions/server/onerep.ts
Comment (flozia) [inline src/app/functions/server/onerep.ts:457]: For [MNTOR-2315](https://mozilla-hub.atlassian.net/browse/MNTOR-2315) we need the total number of scans performed and I guess we could just use this function as well. We could make the `from` prop optional as well and in case no time period is provided just return the total — or do you prefer us explicitly passing `from`.

File: src/app/functions/server/onerep.ts
Comment (mansaj) [reply src/app/functions/server/onerep.ts:456]: Not yet, but as soon as we get the landing page done, we will

File: src/app/functions/server/onerep.ts
Comment (mansaj) [reply src/app/functions/server/onerep.ts:455]: This makes sense, I'll wait till #3779 is merged then

File: src/app/functions/server/onerep.ts
Comment (mansaj) [reply src/app/functions/server/onerep.ts:457]: @flozia ah, that makes sense. I can make the `from` optional as well

Comment (mansaj) [reply src/app/functions/server/onerep.ts]: Fixed

**Review by rhelmer — APPROVED** [review]

**Pull request merged**
